src/doc/conf.py

```diff
@@ -55,17 +55,17 @@
 
 # General information about the project.
 project = u'VisIt User Manual'
-copyright = u'2008-2017, LLNL, UCRL-SM-220449'
+copyright = u'2008-2019, LLNL, UCRL-SM-220449'
 author = u'LLNL'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'2.11'
+#version = u'3.0'
 # The full version, including alpha/beta/rc tags.
-release = u'2.11'
+#release = u'3.0.0b'
 
```

Comment:
i guess sphinx sets these for us, so its safe to leave them out for the not-called-by-cmake  sphinx case?

Reply:
yeah, but I assume from Kathleen that it doesn't get it right on windows and/or for local builds. It only gets it right on RTD builds